README.md

```diff
@@ -4,8 +4,6 @@ This demo includes all of the files necessary to get started with a basic, hello
 
 ## App Installation
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/bigcommerce/sample-app-nodejs)
-
 To get the app running locally, follow these instructions:
 
 1. [Use Node 14+ and NPM 7+](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm#checking-your-version-of-npm-and-node-js)
```

Comment:
With the node upgrade we might want to update this line as well

Reply:
good catch!!